src/index.html

```diff
@@ -9,6 +9,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header__logo" href="#">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-items">
+          <li><a class="header__nav-item is-active" href="#">Apple</a></li>
+          <li><a class="header__nav-item" href="#">Samsung</a></li>
+          <li><a class="header__nav-item" href="#">Smartphones</a></li>
+          <li>
+            <a
+              data-qa="hover"
+              class="header__nav-item new"
```

Comment:
no needs to create such classes
```suggestion
              class="header__nav-item"
```